Comp2023-Imported/src/main/java/frc/robot/subsystems/LED.java

```diff
@@ -57,15 +57,14 @@ public void periodic( )
   @Override
   public void simulationPeriodic( )
   {
-    // This method will be called once per scheduler run when in simulation
-  }
 
-  // Put methods for controlling this subsystem here. Call these from Commands.

```

Comment:
I don't think we should be making any changes to Comp2023-Imported. Could we keep what was there originally?

Reply:
Ditto with what Hannah said. We should back out changes made to Comp2023-imported from this PR.

Reply:
I added it back in since I don't know how to revert changes on one specific file